src/lib/Lib/MrDocsCompilationDatabase.cpp

```diff
@@ -165,6 +167,40 @@ adjustCommandLine(
     return new_cmdline;
 }
 
+std::optional<std::string> 
+executeCmakeExportCompileCommands(llvm::StringRef cmakePath, llvm::StringRef cmakeListsPath) 
+{
+    if ( ! llvm::sys::fs::exists(cmakePath)) {
+        return std::nullopt;
+    }    
+    if ( ! llvm::sys::fs::exists(cmakeListsPath)) {
+        return std::nullopt;
+    }
+
+    llvm::SmallString<128> stdOutPath;
```

Comment:
These outputs are not being used?

Reply:
Not yet, I am not sure if I will use them to return as error messages.